usr/src/man/man1m/zfs.1m

```diff
@@ -3145,31 +3147,35 @@ property.
 The following permissions are available:
 .Bd -literal
 NAME             TYPE           NOTES
-allow            subcommand     Must also have the permission that is being
```

Comment:
This line and above were changed after @yuripv review, formatted output doesn't wrap nicely on an 80 column terminal.